ts/Series/TemperatureMap/TemperatureMapPoint.ts

```diff
@@ -0,0 +1,128 @@
+/* *
+ *
+ *  (c) 2010-2022 Rafal Sebestjanski
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+import type TemperatureMapSeries from './TemperatureMapSeries.js';
+import type TemperatureMapPointOptions from './TemperatureMapPointOptions';
+
+import MapBubblePoint from '../MapBubble/MapBubblePoint.js';
```

Comment:
Pulling series and points directly is forcing the builder to include those parts in the module. Use `SeriesRegistry` as an alternative.